### ISSUE TITLE
Timeshift Scheduled Snapshot in Progress Forever + Eats up HDD

### ISSUE THREAD

**Issue by Areahints** [teejee2008/timeshift#400]
**Describe the bug**
After installing timeshift and setting it to backup everything i had on my system, i restarted and discovered my HDD was full (from about 250GB free) When i try to launch timeshift i get the popup **"another instance of timeshift is creating a snapshot, please wait a few minutes and try again"** i dont know how to find the process and kill it (If you do, tell me)

**To Reproduce**
3. Error message

**Expected behavior**
I should be able to run timeshift and interrupt or modify its behaviour, not wait for it

**Screenshots**

**System:**
 - Linux Distribution Name and Version: [e.g. Ubuntu 18.08]
 - Desktop [XFCE]
 - Application Version [e.g. v18.8]

**Comment by Areahints** [comment]
Putting this here as a guide

`pidof timeshift`

this will display a **PID number** eg 17555

`sudo kill 17555`

when you launch timeshift it should launch correctly

**Comment by datgb** [comment]
Guess this issue is being ignored. Doing the same thing to my install on Parrot OS. Had to kill it to do anything and uninstall timeshift.

**Comment by MahmoudFahiem** [comment]
I don't know you still face the same problem or not. for the question you asked about the difference between users and filters. In users you are limited with user options only, you can't not for example include only the pictures folder. But in filter option you have more options to include or exclude or include any file or folder

**Comment by nowave7** [comment]
Sorry to resurrect a relatively old issue, but I'm afraid this isn't a bug. It comes down to size of what you are trying to back up. I have a fairly large home folder (~120GB) plus a lot of stuff other applications, tools, dev environments, source code, what not. I don't even have any film or music collections, but still the size of the last snapshot went over 200GB alone! And if you choose to do your backups quite often (daily basis, keeping five snapshots), you're in big trouble.

**Comment by rodwgs** [comment]
Are you saving your backups to the same HDD which is being backed up? If so, that's a problem right there; you should backup to a separate partition, at least (ideally a separate drive entirely.)

**Comment by nowave7** [comment]
Hey Rod,

That is indeed the case. The thing is, this came out of the box like that, not the most ideal situation. Not an expert  by any means, but I guess it would be nice to check if that is the only partition/drive present in the system, and ask the user how to set it up, or disable it entirely?

**Comment by rodwgs** [comment]
I agree, the system should ideally try to save to a different partition/drive, even if it has to prompt the user for one. I get that people want one-click-instant-backups right out the gate, but this seems like a necessity.

**Comment by aymswick** [comment]
What I don't understand about this is that the implication is that you can't launch the timeshift GUI while it is running a backup? That seems odd to me and like I'm misunderstanding/have misconfigured something

**Comment by rodjacksonx** [comment]
@aymswick: No, while Timeshift is making a backup, you can't open the GUI window. Also, and you may want to note this, while the GUI window is open, it will perform no backups!

I would presume that this is to ensure no fiddling with the settings occurs during backups, and that if you're in the middle of changing settings, the backups will be on hold until you've completed the changes and saved them. It's a simple, if slightly annoying way of enforcing those restrictions, but it works.

Just don't leave the GUI window open for an extended period, or you could miss some backups.
